ld_script.txt

```diff
@@ -269,6 +269,7 @@ SECTIONS {
         src/battle_factory.o(.text);
         src/battle_pike.o(.text);
         asm/battle_frontier_2.o(.text);
```

Comment:
You forgot to remove battle_frontier_2.asm and its entry in the ld_script.

Reply:
I didn't. There is code there, it has nothing to do with battle frontier though. Mossdeep gym tiles maybe?

Reply:
Maybe you should rename it to something that's not battle_frontier_2 if you're sure it's not frontier-related then.